compiler-plugin/src/test/kotlin/arrow/meta/quotes/scope/IfExpressionTest.kt

```diff
@@ -22,7 +22,11 @@ class IfExpressionTest : AnnotationSpec() {
 
   @Test
   fun `Do not validate if expression without else scope properties`() {
-    // TODO
+    validate(
+      """
+      | if (2 == 3) {
+      |   println("FAKE NEWS")
+      | }""".ifExpression())
```

Comment:
Code examples for `IfExpression` are located in:
```
./compiler-plugin/src/test/kotlin/arrow/meta/quotes/scope/expressions/IfExpression.kt
```

And this comment can be removed with this PR:

```
// TODO: Do not validate if expression without else scope properties
```